CHANGELOG.md

```diff
@@ -27,6 +28,34 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.0.0] - WIP
+
+**NOTE**: This release is currently in alpha and the release notes may change significantly until the first beta release is cut.
+
+#### Added
+
+- support for [UDP][kong-udp] via the new `UDPIngress` API.
+- specifying single and multi-namespace configuration for the controller is now available via the controller manager CLI's `--watch-namespace` flag.
+
+[kong-udp]:https://konghq.com/blog/kong-gateway-2-2-released/#UDP-Support
+
+#### Breaking changes
+
+- running the controller manager via the CLI is now done with a subcommand, instead of `manager <args>` you must call `manager controller <args>` (the arguments thereafter remain the same unless otherwise noted below)
```

Comment:
We should probably fix that. Is it really the case though? If my memory serves me well, there is no longer that subcommand.

Reply:
You're right, this is fixed now but I hadn't realized it :+1: 
```suggestion
```